R/parsed.R

```diff
@@ -9,6 +9,18 @@ compute_parse_data_flat_with_ws <- function(text) {
   parse_data_with_ws
 }
 
+#' Pre-processing parse data
+#'
+#' @details Preprocessing includes
+#' * removing non-terminal entries.
+#' * remove columns id, parent and terminal.
+#' * adding a start token.
+#' * add linebreak and space information.
+#' * replace spaces in comments at the end of the line.
```

Comment:
Okay, that makes sense. We'll need to check if this is still necessary, because I remember code that removes all space at EOL (which isn't safe if a string constant contains an embedded newline, btw.).  For this PR the wording looks good.